### PR TITLE
Handle null secondLanguage in TeacherProfile.vue

### LINKED ISSUE #142: Unable to display Teacher Profile without second language (Frontend)
Cannot display a Teacher without second language (only has native language) on the individual teacher profile page.

### PR DESCRIPTION
**Pull request by therealyingtong**
Closes #142. Recommended to review commit by commit.

Also includes two small cleanups:
- Move `teacherGetters` and `studentGetters` out of `store.js`
- Specify `StudentID` and `TeacherID` as `type: Number`

To test:
1. Create a new teacher with no Second Language.
2. Check that the new teacher's Profile displays correctly.

### CODE REVIEW

File: frontend/src/store.js [hunk 1]
```diff
@@ -76,17 +76,13 @@ export default new Vuex.Store({
...
   getters: {
     ...studentGetters,
     ...nativeLanguageGetters,
     ...teacherGetters,
     ...reasonGetters,
     ...matchesGetters,
-    screeningStudents: (state) => state.students.filter((student) => student.status.Description === "SCREENING"),
-    unmatchedStudents: (state) => state.students.filter((student) => student.status.Description === "UNMATCHED"),
-    getStudentByStudentId: (state) => (id) => state.students.find(student => student.StudentID == id),
-    getTeacherByTeacherId: (state) => (id) => state.teachers.find(teacher => teacher.TeacherID == id)
```
Comment (thchia) [inline frontend/src/store.js:90]: Is this already in the Teachers store file? Because it's not added in this PR. On my phone right now so can't check.

Comment (thchia) [reply frontend/src/store.js:90]: Yeah it is. I just checked.

**Review by thchia — APPROVED** [review]
Untested approval.

**Review by nondescryptid — APPROVED** [review]
Tested, could create and view teacher with no secondlanguage. Getters don't seem to be affected by the reorganisation as components that use student/teacher getters (the "Screening", "Matching" pages are working fine)

**Comment by therealyingtong** [comment]
Force-pushed to resolve conflicts with master.

**Pull request merged**